plugins/rust-openid-token-plugin/src/main.rs

```diff
@@ -59,6 +59,11 @@ pub struct Args {
     /// header field from `api_under_test_header_name`
     #[arg(long, env)]
     api_under_test_header_prefix: Option<HeaderValue>,
+
+    /// Flag for whether or not to overwrite existing headers with the same name
+    #[arg(long, env, action = clap::ArgAction::SetTrue)]
+    overwrite_existing_headers: bool,
```

Comment:
Should this be the default behavior, and "retain existing headers" be the opt-in ?